src/amber/cli/commands/run.cr

```diff
@@ -19,13 +19,15 @@ module Amber::CLI
       end
 
       def run
+        name = Sentry::SentryCommand::Options.get_name
         puts colorize("💎  Crystalizing...", :dark_gray)
-        compile_command = "crystal build $(ls ./src/*.cr | sort -n | head -1) -o app"
+        Dir.mkdir_p("bin")
```

Comment:
Check if it exists before creating it?

Reply:
But `mkdir_p` does essentially the same (checks if exists and creates if not)

Reply:
Yes, it does 👍